### ISSUE TITLE
Drone fails to detect collision whereas the car does in the same environment 

### ISSUE THREAD

**Issue by caesar84** [microsoft/AirSim#4024]
### Sorry for multiple posting as I don't know if this or the question section is more appropriate.
I have imported a costume environment of a street in a city to Unreal engine and the settings were ok. tried to run a drone and after hitting play the drone falls off and goes down the road like collision detection is disabled. ("EnableCollisions": false) . when replacing the drone with the car it never falls even though the player start position is the same and the setting file is unchanged.

#### Context details
Software properties: windows 10
 latest airsim version
 VS2019, 
Unreal 26.4.
<! {
    "SettingsVersion": 1.2,
    "SimMode": "Multirotor",
    "Vehicles": {
        "Drone1": {
          "VehicleType": "SimpleFlight",
          "X": 0, "Y": 0, "Z": 0,
		  "EnableCollisionPassthrogh": false,
          "EnableCollisions": true
        }
        
  }
}
 >

### Include details of what you already did to find answers
As a diagnose, I added the collision detection lines two lines in the settings. json :
 "EnableCollisionPassthrogh": false,
      "EnableCollisions": true,
the car behaves as it should be, however, the drone behaves like "EnableCollisions": false always even if Its  "EnableCollisions": true or not included in the settings.json ( default).

**Comment by caesar84** [comment]
@jonyMarino can you please just have a look at this, is it a bug?

**Comment by jonyMarino** [comment]
Hi @caesar84! You should check the collision settings for the floor inside the editor. @zimmy87 do you remember the exact settings that are needed?

**Comment by caesar84** [comment]
@jonyMarino thank you very much for that note and waiting for @zimmy87 to help , hopefully. Sorry if tagging you was bit inappropriate but I have struggled to sort it out . Should I change the collision settings for the random cars also as the drone fails to detect a collision with them as well?. Finally, does it make sense to you that drone fails while the car detects the collision even it's exactly the same environment? 
Can't thank you enough for your reply .

**Comment by jonyMarino** [comment]
No worries.
Take a look at this comment:
https://github.com/microsoft/AirSim/issues/3855#issuecomment-874984693

That was what i was refering to.

**Comment by caesar84** [comment]
hi @jonyMarino, thank you very much It turns out ground and many buildings have no cillision. thanks alot.

**Comment by Kabilan-n** [comment]
Hi, I'm also facing the same error when I'm trying to use drone i9n custom environment it is falling under the ground and there is no collision with the other buildings also where as when I'm trying with car I was able to get in on the ground and the collisions with buildings are also working properly.

I tried adding collision to the objects and the player (i.e drone) but still it fails to land on the floor. and added { "EnableCollisionPassthrogh": false, "EnableCollisions": true } in my settings.json

This is what I get when I try using car. It was able to land in the ground. 
![car](https://user-images.githubusercontent.com/60337704/228755983-d5038339-4abe-46e1-ab03-a27652cf1f9a.png)

When I try using drone the following happens. 
![drone1](https://user-images.githubusercontent.com/60337704/228774090-936cfc3c-56ec-4ea2-abba-069e047902ab.png)

![drone 2](https://user-images.githubusercontent.com/60337704/228774110-f8000dec-6870-45ee-9062-1b2b2ad2618d.png)
